reference/conanfile/tools/intel_cc.rst

```diff
@@ -0,0 +1,247 @@
+    Using Conan toolchain through .
+    -- Check for working CXX compiler: /opt/intel/oneapi/compiler/2021.3.0/linux/bin/dpcpp -- works
+    -- Detecting CXX compiler ABI info
+    Using Conan toolchain through .
+    -- Detecting CXX compiler ABI info - done
+    -- Detecting CXX compile features
+    -- Detecting CXX compile features - done
+    -- Configuring done
+    -- Generating done
+
+
+As you can see, you have used one of these compilers, the DPC++ one and successfully generated the `libintelcc.a` file.
+
+.. note::
+
+    Conan is running the ``setvars.sh|bat`` automatically in order to enable all the needed environment variables to build the library.
+
+
+Intel oneAPI Toolset and Microsoft Visual Studio
+-------------------------------------------------
+
+.. note::
+
+    Ensure you have installed the Intel plugins for Microsoft Visual Studio before reading this section.
+
+
+If you're working in a Microsoft Visual Studio project, you can add the Intel Toolset as a new *.props* file easily.
+Let's suppose you have defined these files into your current project folder:
+
+.. code-block:: text
+    :caption: intelprofile
+
+    [settings]
+    os=Windows
+    os_build=Windows
+    arch=x86_64
+    arch_build=x86_64
+    compiler=intel-cc
+    compiler.mode=classic
+    compiler.version=2021.3
+    compiler.runtime=dynamic
+    build_type=Release
+    [options]
+    [build_requires]
+    [env]
+    [conf]
+    tools.intel:installation_path="C:\Program Files (x86)\Intel\oneAPI"
+
+
+.. code-block:: python
+    :caption: conanfile.py
+
+    from conans import ConanFile
+    from conan.tools.microsoft import MSBuildToolchain
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+
+        def generate(self):
+            tc = MSBuildToolchain(self)
+            tc.generate()
+
+Running a ``conan install . -pr intelprofile`` you'll see a new *conantoolchain_release_x64.props* file generated in your current folder as the showed bellow:
+
+.. code-block:: xml
+    :caption: conantoolchain_release_x64.props
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+      <ItemDefinitionGroup>
+        <ClCompile>
+          <PreprocessorDefinitions>
+             ;%(PreprocessorDefinitions)
+          </PreprocessorDefinitions>
+          <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
+          <LanguageStandard></LanguageStandard>
+        </ClCompile>
+      </ItemDefinitionGroup>
+      <PropertyGroup Label="Configuration">
+        <PlatformToolset>Intel C++ Compiler 19.2</PlatformToolset>
+      </PropertyGroup>
+    </Project>
+
+Then, you'll be able to load this new properties into your current Microsoft Visual Studio project.
+
+
+Custom definition
+-----------------
+
+Perhaps you want to define/override something else in your own, so you can invoke the environment variables generation from your conanfile.py as follows:
```

Comment:
What environment variables? You have to explain first what the IntelCC does and then what can you adjust (table of methods/attributes)